.vscode/settings.json

```diff
@@ -16,27 +16,16 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
 
-  // Custom file associations
-  "files.associations": {
-    // HACK https://github.com/samuelcolvin/jinjahtml-vscode/issues/47
-    ".env.jinja": "jinja-shell"
-  },
```

Comment:
This issue was solved: https://github.com/samuelcolvin/jinjahtml-vscode/pull/48/files